### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by dancooper-tandem**

### CODE REVIEW

**Review by mattchampion-tandem — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Important Notice
+
+**Do not make this repo private!**
+
+This repository contains essential files that are served by JsDelivr for use by the mortgage intermediary website. Keeping this repository public is crucial for JsDelivr to access the files. Making it private will disrupt the services provided by mortgage intermediary website.
+
+## Why Use A Public Repo?
+
+Our website host only allows externally hosted javascript files, so we are using JsDelivr to host our calculator script.
```
Comment (Ryan-Palmer) [inline README.md:9]: Slight rewording maybe - we are using Github to host it, and JSDelivr to cache / deliver it

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Accessing the Website
+
+To access the mortgage intermediary website and see this calculator in action, please visit [https://www.intermediary.tandem.co.uk/](https://www.intermediary.tandem.co.uk/)
```
Comment (Ryan-Palmer) [inline README.md:13]: Bear in mind this is a public repo and that site has no login. It is fine for anyone to use really but might not want to put the url here (although it is bit late now ;) ) We were just discussing putting a GUID in the path anyway so we can release specific versions to external testers and revoke when needed.

File: README.md (hunk 1 above)
Comment (dancooper-tandem) [reply README.md]: changed :)

File: README.md (hunk 2 above)
Comment (dancooper-tandem) [reply README.md:13]: The intermediary site is not private or secret so it can be left here

**Review by Ryan-Palmer — APPROVED** [review]

**Review by mattchampion-tandem — APPROVED** [review]

**Pull request merged**